bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -587,6 +602,15 @@ def get_arm_joint_positions(self, as_array=True):
             return arm_joint_positions
         return arm_joint_positions.tolist()
 
+    def get_new_goal_given_obj_img(self, depth, bbox):
+        """Estimate the goal location based on the depth and bbox of the target object
+        :param as_array
+        :return the x, y location in the world frame
+        """
+        x = 0
+        y = 0
+        return np.array([x, y])
```

Comment:
Please see the `main()` method from HEAD of https://github.com/facebookresearch/spot-sim2real/tree/experiment/3d_object_detection_with_spotcamera for this implementation @tusharsangam